templates/shared_services/airlock-notifier/.dockerignore

```diff
@@ -0,0 +1,6 @@
+# See https://docs.docker.com/engine/reference/builder/#dockerignore-file
```

Comment:
I have no where else to write this - I think the folder name "airlock-notifier" should be with an underscore.

Reply:
My bad, was going to do that after seeing the other services but forgot

Reply:
Aligned with the other shared services, but I don't mind